tests/serverpod_test_server/test_integration/test_tools/serverpod_test_tools.dart

```diff
@@ -38,23 +38,14 @@ import 'package:serverpod_test_server/src/generated/scopes/scope_server_only_fie
     as _i20;
 import 'package:serverpod_test_server/src/generated/protocol.dart';
 import 'package:serverpod_test_server/src/generated/endpoints.dart';
-export 'package:serverpod_test/serverpod_test.dart'
-    show
-        TestSession,
-        ConnectionClosedException,
-        ServerpodUnauthenticatedException,
-        ServerpodInsufficientAccessException,
-        RollbackDatabase,
-        ResetTestSessions,
-        flushMicrotasks,
-        AuthenticationOverride;
+export 'package:serverpod_test/serverpod_test_public_exports.dart';
 
 @_i1.isTestGroup
 withServerpod(
   String testGroupName,
   _i1.TestClosure<TestEndpoints> testClosure, {
   _i1.ResetTestSessions? resetTestSessions,
-  _i1.RollbackDatabase? rollbackDatabase,
+  _i1.DatabaseTestConfig? databaseTestConfig,
```

Comment:
Question/Discussion: Are we ok with mixing nouns and verbs here?

ResetTestSession described something that was done while databaseTestConfig is more a object we are passing in.

Reply:
Will revert this change as it's out of scope for the PR: https://github.com/serverpod/serverpod/pull/2791#discussion_r1782467043

Reply:
100408c